plan/validator.go

```diff
@@ -46,11 +46,6 @@ type validator struct {
 func (v *validator) Enter(in ast.Node) (out ast.Node, skipChildren bool) {
 	switch node := in.(type) {
 	case *ast.AggregateFuncExpr:
-		if v.inAggregate {
```

Comment:
is it safe to remove these ?

Reply:
This is a grammar check, I'm not sure why we abandon this before.
@shenli PTAL